config/default/manager/manager.yaml

```diff
@@ -158,6 +158,8 @@ webhooks:
         values:
           - "0"
           - "1"
+      matchLabels:
+        mutatepods.kubemacpool.io: all
```

Comment:
What does "all" mean? (looking forward for the docs)

Reply:
the thought was that this label could either be specific to a network name:
mutatepods.kubemacpool.io: mynetwork
or ready for all networks: 
mutatepods.kubemacpool.io: all
or all opted out:
mutatepods.kubemacpool.io: disable
what do you think?

Reply:
I think that there should be additional label on the NetworkAttachmentDefinition. Having list of networks on Namespace is risky since there is no garbage collection and it would require more logic IMO.

I'm ok with `disabled`. `all` should be more specific, maybe `allocateForAll`? For other options, that does not need to be discussed now IMO.

Reply:
ack